source/patterns/@aws-solutions-constructs/aws-events-rule-sns/test/integ.events-rule-no-arg.ts

```diff
@@ -26,6 +26,6 @@ const props: EventsRuleToSnsProps = {
   }
 };
 
-new EventsRuleToSns(stack, 'test-construct', props);
+new EventsRuleToSns(stack, 'test', props);
```

Comment:
Interesting - why do this? Did we collide with something somewhere?

Reply:
Failed member validation because it creates a name longer than 64 characters in the cloud formation template which it doesn't allow